### ISSUE TITLE
Instruction Fail

### ISSUE THREAD

**Issue by Serexx** [pivotal-cf/react-starter#15]
With respect I came here trying to see if I like the how the components work and while I've been coding for a very long time (a multiplicity of languages and platforms) I'm relatively new to Node and SPA etc. so IMHO while the install instructions are verbose the actual step by step guts are both cryptic and make unwarranted assumptions about people's knowledge and setups. 

Actually I ended up here trying react-starter because the "pui-starter-project" project suffers from the same malady, making assumptions about what I know and asking me to copy-paste CSS from somewhere unknown.

TBH I have been using Webpack for everything so the Gulp instructions are useless, as they are in the other project. The "path" reference assumes that I intuitively know whether that is a relative path, a system path, a user path, a path to the starter project or to a required -g install of gulp, or to my installed Node folder, and for that matter, what OS platform I am on.

With a deadline to meet, there simply isn't time to figure it out so I've moved on to evaluating another of the 12 or so other React-targeted UI packages still on my list. Most of those already tested, by the way, have been :

npm install
npm run start

If you want to improve the uptake I'd respectfully suggest that it needs to be a little easier for a broader base of peeps to have a no-hassle look.

Hope this helps improve the entry path :-)

Greg Burman CPA/CGA

**Comment by charleshansen** [comment]
Thanks for the feedback Greg. Sorry this project didn't work for you. I hope you met your deadline :). 

Random question, how did you end up at this repo? Most users seem to be Pivotal employees who are using Pivotal UI and then end up here because of the Pivotal UI getting started instructions (non Pivotal employees who are using Pivotal UI seem to have their own setup already). This might explain some of the disconnect.

There is an implicit assumption in this project that you are probably using OSX and are already generally comfortable with configuring your developer setup. I agree that the readme does not do a very good job for people still learning how to do that stuff. Unfortunately, this project is not a high priority and I don't see the work getting done to create README for a broad base. That said, if you have specific suggestions for words to add to the README, I'll be happy to add them. 

For what it is worth, if you are already using Webpack for everything, you might not need any of the React starter projects. The main job for most of these projects is to hide the Webpack part from you so that you don't have to set it up. Every React Starter project makes significant tradeoffs. If you already know the hard part (Webpack) and aren't on a deadline in the future, you would probably benefit from controlling everything yourself.

I believe more and more developers are becoming comfortable with Webpack now and I am slowly working on a rewrite of this project to stop hiding the Webpack config from you. I hope this will make this a bit easier to understand.

**Comment by Serexx** [comment]
Hi Charles - Got here via old friend Google and then one of several lists of available "best" React U.I. frameworks, sadly I don't recall which list. Given your circumstances (OSX / internal use)  the text makes more sense now. Ironically I started using React with Webpack from the beginning so it actually seems less intimidating to me, but of course I can see the opposite could be true for others. I'm mainly a Windows  back end guy not a designer, thus my search for a good starting point. I was probably a little frustrated when I dropped my note so apologies on that score. I've ended up with Grommet and yes thanks, met my deadline so all is well. 

By the time I get comfy with Node I may not get back to suggest text but I do appreciate your response, and wish you luck. All the best!
